labs/lab08-airgap-lite/README.md

```diff
@@ -1,72 +1,67 @@
 Lab 1.8: Airgap
 =========================================
 
-In this lab, we'll review how to perform installations in Air Gap environments, and
-how to collect support bundles in Air Gap environments.
+In this lab, you will review how to perform installations and collect support bundles in air gap environments.
 
 * **What you will do**:
-    * Access and verify a single-node Air Gap setup via a bastion server
-    * Learn to use KOTS to install in an Air Gap environment
-    * Create an SSH tunnel to configure an Air Gap instance
-    * Perform an upgrade of an application in an Air Gap environment
-    * Use the `kubectl support-bundle` CLI in an Air Gap environment
-* **Who this is for**: This lab is for anyone who builds/maintains KOTS applications (see note below)
+    * Access and verify a single node air gap setup using a bastion server
+    * Learn to use app manager to install in an air gap environment
```

Comment:
Since this is for a more general audience 

```suggestion
    * Learn to use the [kURL.sh](https://kurl.sh) Kubernetes Installer in an air gap environment
    * Learn to use app manager to install in an air gap environment
```